pages/meta.json

```
@@ -10,5 +10,6 @@
   "voting": "veFLOW Voting",
   "security": "Security",
   "c4report": "C4 Report",
-  "legal": "Legal Disclaimer"
-}
\ No newline at end of file
+  "legal": "Legal Disclaimer",
+  "pressKit": "Press Kit",
```

delete last ',' as it's not a js, it's json
```suggestion
  "pressKit": "Press Kit"
```